base/test.jl

```diff
@@ -120,6 +127,24 @@ function Base.show(io::IO, t::Error)
     end
 end
 
+"""
+    Broken
+
+The test condition either has been explicitly skipped or is the expected
```

Comment:
Missing a genstdlib run, and I don't think `test_skip` should be exclusively for intermittent failures. I'll take a pass at it later if I don't forget.